flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/BackendRow.java

```diff
@@ -25,44 +25,49 @@
 public class BackendRow {
 
     @JsonProperty(value = "HttpPort")
-    private String HttpPort;
+    private String httpPort;
 
     @JsonProperty(value = "IP")
-    private String IP;
+    private String ip;
 
     @JsonProperty(value = "Alive")
-    private Boolean Alive;
+    private Boolean alive;
 
     public String getHttpPort() {
-        return HttpPort;
+        return httpPort;
```

Comment:
Will there be a problem if change it to lowercase here?

Reply:
The main function here is to map the String type to a Java object through Json. After declaring the mapped field type that needs to be converted through @JsonProperty(value = "xxx"), there is no problem here.